pkg/server/server_integration_test.go

```diff
@@ -37,15 +37,50 @@ func TestSendingMessagesBetweenPeers(t *testing.T) {
 	ws2 := peerJoinsRoom(t, httpServer, room, peerSecret2)
 	defer ws2.Close()
 
+	readMessage(t, ws1) // skip 'peer_connected'
```

Comment:
Did you run test.sh so that linters are ran? I'm wondering if you can just ignore the return value or you need to do _ = readMessage. I thought you have to and linter should scream. Does it happen?